docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -391,6 +391,24 @@ This optimization applies to all forms of bean declarations: bean class, produce
 Users can instruct the container to not remove any of their specific beans (even if they satisfy all the rules specified above) by annotating them with `io.quarkus.arc.Unremovable`.
 This annotation can be placed on the types, producer methods, and producer fields.
 
+Since this is not always possible, there is an option to achieve the same via `application.properties`.
+The `quarkus.arc.unremovable-types` property accepts a list of string values that are used to match beans based on their simple name or package.
```

Comment:
This is not only the simple name but also the fully qualified one. Maybe drop 'simple' ?

Reply:
Good point, it is unclear. You can actually use FQCN but also just simple class name. E.g. you can use `org.foo.Bar` or just `Bar`.

Reply:
Removed the word `simple`